packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -41,15 +41,14 @@ export const styles = theme => ({
     overflowX: 'hidden',
     width: '100%',
   },
-  /* Styles applied to the tablist element and its hidden scrollbar vendor prefixed if `variant="scrollable"`. */
+  /* Styles applied to the tablist element if `variant="scrollable"`. */
   scrollable: {
     overflowX: 'scroll',
-    msOverflowStyle: 'none',
-    overflow: '-moz-scrollbars-none',
-    scrollbarWidth: 'none',
+    // Hide dimensionless scrollbar on MacOS
```

Comment:
Do we need to write this style when `this.state.scrollerStyle.marginBottom === 0`? As this will hide scrollbar on Windows also but won't create an issue

Reply:
This is only necessary for MacOS when the scrollbar is configured to have no dimension. We don't need it on Windows.

Reply:
I am sorry, tell me if I am correct. Right now the code is still running for Windows. To make it run only on MacOS I think we need to check if scroll Dimension is 0 using this check `this.state.scrollerStyle.marginBottom === 0`

Reply:
Why is this a problem?